src/main/java/com/devathon/preguntonic/dto/PlayerStatusDto.java

```diff
@@ -0,0 +1,44 @@
+package com.devathon.preguntonic.dto;
+
+import java.util.List;
+import java.util.UUID;
+
+import com.devathon.preguntonic.model.Player;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.Builder;
+
+@Builder
+public class PlayerStatusDto {
+        
+    @JsonProperty("id")
+    private UUID id;
+
+    @JsonProperty("nickname")
+    private String nickname;
```

Comment:
please confirm with front-end if we are really going to use the nickname attribute here or just name.
@NicolasLLans @Alejandro25AR @WanderleeDev

Reply:
Only playername.